great_expectations/expectations/expectation.py

```diff
@@ -203,6 +203,29 @@ def _validate(
     ) -> Union[ExpectationValidationResult, dict]:
         raise NotImplementedError
 
+    @classmethod
+    @renderer(renderer_type="atomic.prescriptive.kwargs")
+    def _prescriptive_kwargs(
```

Comment:
This is now the default atomic prescriptive renderer. It will only be used in cases where all other renderers for a given expectation (or custom expectation) are 1. missing atomic renderers and 2. fail when attempting to use the preferred default renderer `atomic.prescriptive.summary`.